festim/exports/surface_flux.py

```diff
@@ -0,0 +1,53 @@
+from dolfinx import fem
+import festim as F
+import ufl
+import csv
+
+
+class SurfaceFlux(F.SurfaceQuantity):
+    """Exports surface flux at a given subdomain
+
+    Args:
+        field (festim.Species): species for which the surface flux is computed
+        surface_subdomain (festim.SurfaceSubdomain1D): surface subdomain
+        filename (str, optional): name of the file to which the surface flux is exported
+
+    Attributes:
+        field (festim.Species): species for which the surface flux is computed
+        surface_subdomain (festim.SurfaceSubdomain1D): surface subdomain
+        filename (str): name of the file to which the surface flux is exported
+        t (list): list of time values
+        data (list): list of surface flux values
+        title (str): title of the exported data
+        write_to_file (bool): True if the data is exported to a file
+    """
+
+    def __init__(
+        self,
+        field,
+        surface_subdomain: int or F.SurfaceSubdomain1D,
```

Comment:
Here this argument cannot be an `int` since the `id` attribute is used for setting `self.title` just below.